bundles/org.openhab.binding.freeboxos/README.md

```diff
@@ -0,0 +1,235 @@
+
+## Supported Things
+
+This binding supports the following thing types:
+
+| Thing         | Thing Type | Description                                             |
+|---------------|------------|---------------------------------------------------------|
+| revolution    | Bridge     | A Freebox Revolution server                             |
+| delta         | Bridge     | A Freebox Delta server                                  |
+| phone         | Thing      | The phone wired to the Freebox Server.                  |
+| host          | Thing      | A network device on the local network.                  |
+| airplay       | Thing      | An AirPlay device in the local network.                 |
+
+## Discovery
+
+The Freebox Server is discovered automatically through mDNS in the local network.
+After a Freebox Server is discovered and available to openHAB, the binding will automatically discover phone, network devices / interfaces and AirPlay devices with video capability in the local network.
+Note that the discovered thing will be setup to use only HTTP API (and not HTTPS) because only an IP can be automatically determined while a domain name is required to use HTTPS with a valid certificate.
+
+## Binding configuration
+
+The binding has the following configuration options, which can be set for "binding:freebox":
+
+| Parameter               | Name         | Description                                                                | Required |
+|-------------------------|--------------|----------------------------------------------------------------------------|----------|
+| callbackUrl             | Callback URL | URL to use for playing notification sounds, e.g. <http://192.168.0.2:8080> | no       |
+
+## Thing Configuration
+
+### Server : Revolution or Delta
+
+The *revolution* or *delta* bridge thing requires the following configuration parameters:
+
+| Parameter Label                    | Parameter ID            | Description                                                                 | Required | Default              |
+|------------------------------------|-------------------------|-----------------------------------------------------------------------------|----------|----------------------|
+| Freebox Network Address            | fqdn                    | The IP address / FQDN of the Freebox Server (can include port number).      | false    | mafreebox.freebox.fr |
+| Application token                  | appToken                | Token generated by the Freebox Server.                                      | false    |                      |
+| Refresh Interval                   | refreshInterval         | The refresh interval in seconds which is used to poll given Freebox Server. | false    | 30                   |
+| Use only HTTP API                  | useOnlyHttp             | Use HTTP API even if HTTPS is available.                                    | false    | false                |
```

Comment:
I would probably change the Required column to use Yes and No so it cant be confused with the true and false of the default column.

Reply:
Done